RetrospectiveExtension.Frontend/config/environment.tsx

```diff
@@ -1,7 +1,7 @@
 const environment = {
-  CollaborationStateServiceUrl: '<collaboration state service url>',
+  CollaborationStateServiceUrl: 'https://my-backend-service.com', // change this to the deployed backend service
```

Comment:
Although `enviornment.tsx`  is gitignored, the file is still needed by the code to compile and the CI/CD tests depend on it. So I left the file as is, but it is still gitignored.